opm/parser/eclipse/EclipseState/Schedule/UDQ/UDQDefine.hpp

```diff
@@ -31,6 +31,7 @@
 #include <opm/parser/eclipse/EclipseState/Schedule/UDQ/UDQContext.hpp>
 #include <opm/parser/eclipse/EclipseState/Schedule/UDQ/UDQFunctionTable.hpp>
 #include <opm/common/OpmLog/KeywordLocation.hpp>
+#include <src/opm/parser/eclipse/EclipseState/Schedule/UDQ/UDQToken.hpp>
```

Comment:
The header files in the source directory are not installed; if we need to access this header file from here it should be moved out of the `src/`directory and added to the list of public headers in `CMakeLists_files`

Reply:
I did not find a solution without accessing the header file. Hence, as suggested, it is moved and added to the list of public headers.